worker/games.py

```diff
@@ -70,6 +71,12 @@ def is_64bit():
 MAKE_CMD = "make COMP=mingw " if IS_WINDOWS else "make COMP=gcc "
 
 
+def log(s):
+    logfile = os.path.join(os.path.dirname(os.path.realpath(__file__)), LOGFILE)
+    with open(logfile, "a") as f:
+        f.write("{}: {}\n".format(datetime.datetime.utcnow(),s))
```

Comment:
If I want to graph directly with the timestamp having `{} : {}\n` is an advantage